.clang-format

```diff
@@ -4,7 +4,7 @@
 ## See: https://clang.llvm.org/docs/ClangFormatStyleOptions.html for details of each option
 ##
 ## The clang-format binaries can be downloaded as part of the clang binary distributions
-## from http://releases.llvm.org/download.html
+## from https://releases.llvm.org/download.html
 ##
 ## Use the script Utilities/Maintenance/clang-format.bash to faciliate
 ## maintaining a consistent code style.
```

Comment:
Did you intentionally modify [Documents/Design/GeneralFrameworkOverview.odp](https://github.com/InsightSoftwareConsortium/LesionSizingToolkit/pull/44/commits/8c951d48b76fe106be650f85c0566e74b8feb2d5#diff-82298e917030cec912100c92f1db9b63c5e61741fcafcdc8cdf7030550214f2b)?

Reply:
No, good catch. I saw a handful of cases in other remote modules where binary files were modified for unknown reasons. I was typically able to pick these out on visual review and reset them, but this one slipped through the cracks. I will push a fix.